hack/testing/check-EFK-running.sh

```diff
@@ -30,8 +30,8 @@ if [ "$USE_OPERATOR" = true ] ; then
 		oal_expected_deployments=( "kibana-app" "kibana-infra" )
 		oal_expected_cronjobs=( "curator-app" "curator-infra" )
 		oal_expected_routes=( "kibana-app" "kibana-infra" )
-		oal_expected_services=( kibana-app kibana-infra cluster-logging-operator elasticsearch-app \
-			elasticsearch-app-cluster elasticsearch-infra elasticsearch-infra-cluster elasticsearch-operator )
+		oal_expected_services=( kibana-app kibana-infra elasticsearch-app \
```

Comment:
are 'app' and 'infra' still expected given we don't support "split" deployments yet

Reply:
no - but - there are many other places in the test where we handle the split deployment - I'd like to merge this PR first, then go back and remove the split deployment handling from all of the tests, and fluentd configs, and elasticsearch configs, etc. etc.